kernel/rtlil.h

```diff
@@ -715,6 +715,9 @@ struct RTLIL::Const
 	std::string as_string(std::string any = "-") const;
 	static Const from_string(const std::string &str);
 	std::vector<RTLIL::State> to_bits() const;
+	std::string pretty_fmt() const;
+	std::string pretty_fmt_undef() const;
```

Comment:
These helpers both have a single call site, it's not obvious what they do, I suggest we remove them